collections/src/main.rs

```diff
@@ -41,17 +46,19 @@ fn list_med(list: &[i32]) -> ListResult {
                         max = Some((key, val))
                     }
                 }
-                None => max = Some((key, val))
+                None => max = Some((key, val)) // FIXME
```

Comment:
The code above is too complicated for such simple task. Consider easier solution.